src/components/Header.js

```diff
@@ -1,53 +1,62 @@
 import React from "react"
 import { Nav, NavItem } from "reactstrap"
 import { NavLink } from "react-router-dom"
-import DogImage from "../assets/DogHeader.jpg"
-
-
+import DogImage from "../assets/headerimage.jpg"
 
 const Header = () => {
     return (
     <>
+      {/* <div className="header-background"> */}
         <Nav className="header-nav">
-      <NavItem>
+      {/* <NavItem>
         <NavLink to="/">
-          <img
+        <img
             src={DogImage}
             alt="header"
-            className="dog-logo"
+            className="dog -logo"
```

Comment:
Best practice for naming css selector names is not to have a space to maintain consistency. This will require changing on this file and the css file.